index.rst

```diff
@@ -0,0 +1,190 @@
+..  Content of technical report.
+
+  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
+  for a guide to reStructuredText writing.
+
+  Do not put the title, authors or other metadata in this document;
+  those are automatically added.
+
+  Use the following syntax for sections:
+
+  Sections
+  ========
+
+  and
+
+  Subsections
+  -----------
+
+  and
+
+  Subsubsections
+  ^^^^^^^^^^^^^^
+
+  To add images, add the image file (png, svg or jpeg preferred) to the
+  _static/ directory. The reST syntax for adding the image is
+
+  .. figure:: /_static/filename.ext
+     :name: fig-label
+     :target: http://target.link/url
+
+     Caption text.
+
+   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
+   See the README at https://github.com/lsst-sqre/lsst-report-bootstrap or
+   this repo's README for more info.
+
+   Feel free to delete this instructional comment.
+
+:tocdepth: 1
+
+This document summarizes the work that was done to test the accuracy of shear measurement using the measurement algorithms in the DM package meas_modelfit.git.  This package contains the Python package lsst.meas.modelfit, which houses both the CModel algorithm -- used to measure galaxy shapes, and the ShapeletPsfApprox Algorithm (SPA) -- used to create a parameterized model of the point spread function (Psf).
+
+The images used for this test were created by GalSim simulations as described below. The galaxy profiles areies parameterized GalSim models. However, the Psfs are not from GalSim models, as we want the framework to accommodate a variety of different realistic and simulated Psfs. A separate psf image for each galaxy is used to simulate the effects of seeing:  this includes both atmospheric and optical effects. 
```

Comment:
I think the only context in which "GalSim models" for PSFs should be mentioned would be if you wanted to draw a contrast with the GREAT3 simulations, e.g.:
> The images used for this test were created by GalSim simulations as described below.  We ran GalSim using a modification of the scripts used for the GREAT3 project, modified to use PSF images produced by PhoSim instead of GalSim's optical/atmospheric PSF models.